src/opentimelineio/CMakeLists.txt

```diff
@@ -78,7 +78,7 @@ add_library(opentimelineio ${OTIO_SHARED_OR_STATIC_LIB}
 add_library(OTIO::opentimelineio ALIAS opentimelineio)
 
 target_include_directories(opentimelineio
-    PRIVATE       "${IMATH_INCLUDES}"
+    PUBLIC        "${IMATH_INCLUDES}"
```

Comment:
Thanks for spotting this issue. This change makes sense in order to be able to use otio as a subproject. The reason it is PRIVATE currently is that the deps directory, and optional-lite directory are exposed, without checking first to see if the versions of the libraries in `deps` are used, or if the user has externally supplied their own dependent libraries. Some more work would be necessary to make the directory exports safe for general use.

Reply:
In other words, the includes should be added via variables, such as `${OPTIONAL_LITE_INCLUDES}` or `${RAPIDJSON_INCLUDES}`. That would be better in any case...